.github/workflows/xcube_workflow.yaml

```diff
@@ -86,13 +86,21 @@ jobs:
     needs: build-docker-image
     name: update-xcube-tag
     steps:
-      - name: git-checkout
-        uses: actions/checkout@v2
-      - uses: actions/checkout@v2
+      - name: Get installation token
+        id: get_installation_token
+        uses: tibdex/github-app-token@v1
         with:
+          app_id: ${{ secrets.TOKEN_PROVIDER_APP_ID }}
+          private_key: ${{ secrets.TOKEN_PROVIDER_KEY }}
           repository: bc-org/k8s-configs
-          token: ${{ secrets.API_TOKEN_GITHUB }}
-          path: k8s
+          # the installationId of the GitHub app we are using
+          installationId: 36950178
```

Comment:
I'd prefer `installationId` to be a secret too -- not sure whether it's actually sensitive, but seems to me like the kind of variable information that doesn't belong in version control. But I don't insist on it.